harvester_e2e_tests/integration/test_vm_functions.py

```diff
@@ -101,14 +153,414 @@ def test_minimal_vm(api_client, image_id, unique_vm_name, wait_timeout):
 
 @pytest.mark.p0
 @pytest.mark.virtualmachines
-@pytest.mark.dependency(depends=["minimal_vm"])
+@pytest.mark.negative
+@pytest.mark.parametrize("resource", [dict(cpu=MAX), dict(mem=MAX), dict(disk=MAX),
+                                      dict(mem=MAX, cpu=MAX), dict(mem=MAX, cpu=MAX, disk=MAX)],
+                         ids=['cpu', 'mem', 'disk', 'mem-and-cpu', 'mem-cpu-and-disk'])
+def test_create_vm_no_available_resources(request, resource, api_client, image_id,
```

Comment:
fixture `request` not used.

Reply:
great catch :smile:  - I've removed the unused pytest request fixture :smile: